tests/test_properties.py

```diff
@@ -152,6 +152,101 @@ def test_multiple_identical_docprops_get_updated(self):
                 'docprop {} was not updated'.format(i+1)
 
 
+class TestRemoveField(object):
+
+    def test_removes_simple_field_but_keeps_value(self):
+        document = Document(docx_path('outdated_docproperty_with_umlauts.docx'))
+        assert 1 == len(document.paragraphs), 'input file should contain 1 paragraph'
+        fields = xpath(
+            document.element.body,
+            u'.//w:fldSimple[contains(@w:instr, \'DOCPROPERTY "F\xfc\xfc"\')]//w:t')
+        assert 1 == len(fields), 'should contain one simple field docproperty'
+
+        assert u'Hie chund ds property: ' == document.paragraphs[0].text
+        assert u'xxx' == fields[0].text
+
+        CustomProperties(document).remove_field(u"F\xfc\xfc")
```

Comment:
Just as an FYI remark, instead of `remove_field` this method could also be called `dissolve_field`.